### ISSUE TITLE
Be able to custom the display when no failed builds.

### ISSUE THREAD

**Issue by Nguonchhay** [jan-molak/jenkins-build-monitor-plugin#289]
Currently, the plugin display only the configure background. In my case, I want to display custom view like `Yeah, all builds are passed.` with green background.

**Comment by jan-molak** [comment]
I agree, that would be so cool! Something along the lines of "Everything works, you're a star!"

This functionality does not exist yet, but I'm more than happy to help with reviewing and merging in a pull request.

J

**Comment by visay** [comment]
+1 to have this feature
